db/migrate/20170719125120_remove_confirmable_from_enterprises.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveConfirmableFromEnterprises < ActiveRecord::Migration
+  def up
+    remove_columns :enterprises, :confirmation_token, :confirmed_at, :confirmation_sent_at, :unconfirmed_email
```

Comment:
This change was not reflected in `db/schema.rb`. I amended this commit and also fixed it up with 18b98d3c5dbe602cf8f4f914744d9239817ab5b8 at the same time.